dspace-api/src/main/java/org/dspace/submit/step/UploadStep.java

```diff
@@ -614,7 +614,7 @@ public int processUploadFile(Context context, HttpServletRequest request,
                 if (configurationService.getBooleanProperty("submission-curation.virus-scan"))
                 {
                     Curator curator = new Curator();
-                    curator.addTask("vscan").curate(item);
+                    curator.addTask("vscan").curate(context, item.getHandle());
```

Comment:
Indeed, see line 59.  Maybe the code in UploadStep just missed being refitted to use services.